controller/src/main.rs

```diff
@@ -35,25 +41,19 @@ impl SchedulingService for SchedulerService {
                 WorkloadStatus {
                     name: "Workload 1".to_string(),
                     status_code: 0,
-                    message: "Workload 1 is running".to_string(),
+                    message: "Your workload is WAITING".to_string(),
                     ..Default::default()
                 },
                 WorkloadStatus {
                     name: "Workload 1".to_string(),
-                    status_code: 0,
-                    message: "Workload 1 is terminated".to_string(),
-                    ..Default::default()
-                },
-                WorkloadStatus {
-                    name: "Workload 2".to_string(),
-                    status_code: 0,
-                    message: "Workload 2 is running".to_string(),
+                    status_code: 1,
+                    message: "Your workload is RUNNING".to_string(),
```

Comment:
FWIW, it's not really useful to pass a string message back as the status. The status should have all necessary information for the caller/receiver to build any logging message it wants to display. Something to be fixed later.

Reply:
this message is based on the protobug from the scheduler team we will have to check with them to remove it there